miner/Miner.hs

```diff
@@ -374,7 +374,7 @@ gpu (GPUEnv mpath margs) (TargetBytes target) (HeaderBytes blockbytes) = do
           throwString err
       Right (MiningResult nonceBytes numNonces hps _) -> do
           let newBytes = nonceBytes <> B.drop 8 blockbytes
-              secs = numNonces `div` hps
+              secs = numNonces `div` max 1 hps
```

Comment:
Is this right?  Dividing `numNonces` by `hps` doesn't seem like the right thing.

Reply:
That's how you get the seconds back out.

Reply:
Oh, disregard.  This is `secs`.